story.css

```diff
@@ -0,0 +1,47 @@
+ul
+{
```

Comment:
General style of defining a style  will have the opening flower bracket next to the style name.

Reply:
in VSCode code formatter is available. Use that.

Reply:
Rearranged  the flower brackets in css file.